docs/elementExtensions.md

```diff
@@ -83,18 +88,18 @@ def my_cool_function():
 
 ### Extension Dependencies
 
-Similar to how questions and elements may require client-side assets, extensions may also require client-side JavaScript and CSS. The different properties are summarized here:
+Similar to how questions and elements may require client-side assets, extensions may also require client-side JavaScript and CSS. The different properties are summarized here. Note that script dependencies may be set as either static or dynamic dependencies, while styles may only be set as static dependencies.
```

Comment:
Consider linking to the `devElements` page, which contains valuable information about how to actually use dynamic dependencies.